src/js/base/app.js

```diff
@@ -1,6 +1,8 @@
-import { bind, isArray, noop, uniqueId, isError } from 'underscore';
+import { bind, isArray, noop, uniqueId } from 'underscore';
 import { App } from 'marionette.toolkit';
 
+import handleErrors from 'js/utils/handle-errors';
```

Comment:
Currently only used in one place, but I moved it to a util, because we could potentially use it elsewhere and I can unit test it there.